elastic_connect/base_model.py

```diff
@@ -1,7 +1,5 @@
 import elastic_connect
-from .join import SingleJoin, MultiJoin
-import datetime
-import dateutil.parser
+import elastic_connect.data_types as data_types
 
 
 class Model(object):
```

Comment:
Dodelat (overit) moznost pouzivat "slots".
Zjistit, jestli to umoznuje naseptavani v pycharmu (ze objekt ma tyhle a tyhle sloty)
Napsat na to test (ve kterem se pouzije model se slots, a musi fungovat stejne jako bez nich)